### PR TITLE
highcharts/2-polar-rings

### PR DESCRIPTION
**Pull request by Xillez**

### CODE REVIEW

File: highcharts-api/highcharts/1-simple-column/main.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+ * @param {number} max - Max value to configure for.
+ */
+function configureZoomAndPlotline(chart, max) {
+    chart.update({
+        yAxis: {
+            max: 2 * max
+        }
+    });
+
+    chart.yAxis[0].addPlotLine({
```
Comment (magdalena-gut) [inline highcharts-api/highcharts/1-simple-column/main.js:47]: You've only used one way to draw the lines. Try finding two more ways to do achieve that. Let me know if you will need any tips!

Comment (Xillez) [reply highcharts-api/highcharts/1-simple-column/main.js]: Tried to add all methods I could find although I'm a little uncertain if the removePlotLine+addPlotLine for the maximum indicator actually count as one 😅 

I tried to add it through the yAxis config but didn't really see a clear and neat way of updating them, and given update() overwrites them, I have to also re-add it afterwards, which isn't really all that pretty 😅

Comment (magdalena-gut) [reply highcharts-api/highcharts/1-simple-column/main.js]: Yes, in practice, that solution might not make much sense 😅 However, we can consider it one of the solutions as you practised load event and update method  🙌 
You were mainly focused on adding plot lines in this task, but it's possible to add lines (circles) not only through plotLines but also by adding a new series or using the SVGRenderer. The latter approach is quite important in Highcharts, so I'd like you to practice it a bit. Could you please create a responsive circle on the chart using the SVG renderer? You can refer to the Highcharts.SVGRenderer documentation here: https://api.highcharts.com/class-reference/Highcharts.SVGRenderer"

Comment (Xillez) [reply highcharts-api/highcharts/1-simple-column/main.js]: Aha I see, will do! 👍

**Review by magdalena-gut — CHANGES_REQUESTED** [review]
Great, nice way of positioning the circle relative to the chart.  Just one more nitpick here. When we resize the chart, the circle is moving. The `load `event allows to manipulation of the chart only once, when the chart is finished loading.  Try to use chart `render `event (Fires after the initial load of the chart and after each redraw) and make the circle responsive (The r should resize with resizing chart. You will face the multiplication of the circles each time when resizing, which you can resolve different ways).
Quick overview of the API:
https://api.highcharts.com/highcharts/chart.events.load
https://api.highcharts.com/highcharts/chart.events.redraw - just FYI
https://api.highcharts.com/highcharts/chart.events.render
The nice demo that shows the differences between these three:
https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/chart/events-render/

File: highcharts-api/highcharts/2-polar-rings/main.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                });
+
+                yAxis.addPlotLine({
+                    id: '1.75*max',
+                    value: 1.75 * max,
+                    dashStyle: 'Solid',
+                    color: '#FD7E7E'
+                });
+            },
+            render: function () {
```
Comment (magdalena-gut) [inline highcharts-api/highcharts/2-polar-rings/main.js:44]: JFYI, another approach could be prioritize encapsulation and isolation when rendering the circle:
```
    events: {
      render: function() {
        const chart = this,
          yAxis = this.yAxis[0],
          max = yAxis.dataMax,
          cx = chart.plotLeft + chart.pane[0].center[0],
          cy = chart.plotTop + chart.pane[0].center[1];
        if (!chart.circle) {
          chart.circle = chart.renderer.circle().attr({
            fill: 'none',
            stroke: '#43BDFF6C',
            'stroke-width': 1,
            'stroke-dasharray': '5,5'
          }).add();
        }
        chart.circle.attr({
          cx: cx,
          cy: cy,
          r: yAxis.len * (max / yAxis.max),
        });
      }
    }
```

**Review by magdalena-gut — APPROVED** [review]
Nicely done, good job!💪

File: highcharts-api/highcharts/2-polar-rings/main.js (hunk 2 above)
Comment (Xillez) [reply highcharts-api/highcharts/2-polar-rings/main.js:44]: True, thanks for the tip 👍

**Pull request merged**